smart_card_connector_app/src/window-logs-exporting.js

```diff
@@ -81,20 +84,20 @@ function exportLogs() {
       ' log messages from the log buffer');
   var copyingSuccess = GSC.Clipboard.copyToClipboard(dumpedLogs);
 
-  goog.dom.classlist.remove(
-      exportLogsElement, EXPORT_LOGS_ELEMENT_EXPORTING_CLASS);
+  exportLogsElement.innerText =
```

Comment:
nit: Code would look a bit cleaner if this would be put inside "else" branch of the condition below.